client/src/components/Print/layouts/singleDay.tsx

```diff
@@ -0,0 +1,336 @@
+import React, { useMemo } from "react";
+import { PrintLayout } from "./types";
+import { Temporal } from "@js-temporal/polyfill";
+import {
+  useProgramDates,
+  useGroups,
+  usePkgs,
+  usePrograms,
+} from "../../TimetableV2/hooks";
+import { epochMillisecondsToPlainDateTime } from "../../TimetableV2/utils";
+import { ScheduledProgram } from "../../TimetableV2/types";
+
+interface SingleDayOptions {
+  date?: Temporal.PlainDate;
+  boldPkgs: (string | null)[];
+}
+
+type SingleDayValidOptions = SingleDayOptions;
+
+const timetable: PrintLayout<SingleDayOptions, SingleDayValidOptions> = {
+  label: "Na den",
+  initialOptions: {
+    date: undefined,
+    boldPkgs: [],
+  },
+  OptionsComponent: ({ options, setOptions }) => {
+    const days = useProgramDates();
+    const packages = usePkgs();
+    const day = options.date;
+
+    return (
+      <>
+        <div className="mb-3">
+          <label htmlFor="day">Den:</label>
+
+          <div className="d-flex flex-row gap-3 flex-wrap">
+            <div className="form-check">
+              <input
+                type="radio"
+                name="day"
+                id="allDays"
+                value="allDays"
```

Comment:
`value` prop should probably not be used on radio or checkbox, just the `checked` prop should be enough